CONTRIBUTING.rst

```diff
@@ -29,86 +29,80 @@ Documentation Convention
 
 Our documentation project setup can be found in the `README. <https://github.com/globalwordnet/gwadoc/blob/master/README.md>`_
 
-Start reading our documentation and you’ll get the hang of it. We optimize for readability.
-We indent using two spaces.
+Most of our contributions happen in the ``doc_en.py`` file and we generate the html files from this automatically.
 
-Most of our contributions happen in the ``doc_en.py`` file and we
-generate the html files from this automatically.
+When writing please consider your readers, we would like the documentation reader friendly for beginners as well as useful for experts.
 
-When writing please consider your readers, we would like the
-documentation reader friendly for beginners as well as useful for
-experts.
+How to add a new relation
+=========================
+
+Documentation for a new relation should be done after discussing the relation with generally.
+
+- Add an issue suggesting what should be added
+- Add the relations to RELATIONS in ``inventories.py``
+- Add basic information about the relations to ``doc_basic.py``
+- Add documentation to (at least) ``doc_en.py``
+- Check the project doc folder to have access to useful content in the `Euro Wordnet General Documentation <https://globalwordnet.github.io/gwadoc/pdf/EWN_general.pdf>`_ and `Piasecki_Wordnet <https://globalwordnet.github.io/gwadoc/pdf/ Piasecki_Wordnet.pdf>`_  pdf documents.
```

Comment:
First, how about "Check the `docs/pdf/` folder...", but also how does having "access to useful content" in these files help with adding a relation? That part isn't clear.

Reply:
Corrected